RNA-Seq Coverage/RNA-Seq Coverage.ipynb

```diff
@@ -100,9 +96,7 @@
   {
    "cell_type": "code",
    "execution_count": 3,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "using Bio.Align"
```

Comment:
The current package is [BioAlignments](https://github.com/BioJulia/BioAlignments.jl) v1.